TestVectors/proof-set-chain/multiKeyPairs.json

```
@@ -0,0 +1,18 @@
+{
+  "keyPair1": {
+    "publicKeyMultibase": "z6MktgKTsu1QhX6QPbyqG6geXdw6FQCZBPq7uQpieWbiQiG7",
+    "privateKeyMultibase": "z3u2W4YnTstS1nSSBAgZcYSJF43JuZ9uLV6bF38B1Bf8NugW"
```

IIRC, these should all be `secretKeyMultibase`:

```suggestion
    "secretKeyMultibase": "z3u2W4YnTstS1nSSBAgZcYSJF43JuZ9uLV6bF38B1Bf8NugW"
```

---

Did a quick survey. Wikipedia (https://en.wikipedia.org/wiki/Public-key_cryptography) for public key cryptography uses the terms public and private key. Multi-codecs uses the terms public and private for the different parts of a key pair. Boneh and Shoup (https://crypto.stanford.edu/~dabo/cryptobook/BonehShoup_0_6.pdf) use the terms public and secret key. But I think that's so they can use terse notation such as *sk* and *pk*. All the test vectors in this document use the public/private terminology. Since both are common in the literature this should not cause confusion and I'd prefer not to change all the test vectors.

---

The Data Integrity specifications use the term `secretKeyMultibase`, as defined here: https://w3c.github.io/vc-data-integrity/#multikey